importd/__init__.py

```diff
@@ -268,6 +274,14 @@ def __init__(self):
         """Init class."""
         self.blueprint_list = []
 
+    def openenv(self):
```

Comment:
We have to allow path too. It should accept both relative and absolute path. If its relative, its with respect to calling file.